docs/content/en/docs-dev/user-guide/managing-piped/configuration-reference.md

```diff
@@ -82,57 +82,7 @@ spec:
-
-## CloudProviderConfig
-
-Must be one of the following structs:
-
-### CloudProviderKubernetesConfig
-
-| Field | Type | Description | Required |
-|-|-|-|-|
-| masterURL | string | The master URL of the kubernetes cluster. Empty means in-cluster. | No |
-| kubeConfigPath | string | The path to the kubeconfig file. Empty means in-cluster. | No |
-| appStateInformer | [KubernetesAppStateInformer](#kubernetesappstateinformer) | Configuration for application resource informer. | No |
-
-### CloudProviderTerraformConfig
-
-| Field | Type | Description | Required |
-|-|-|-|-|
-| vars | []string | List of variables that will be set directly on terraform commands with `-var` flag. The variable must be formatted by `key=value`. | No |
-
-### CloudProviderCloudRunConfig
-
-| Field | Type | Description | Required |
-|-|-|-|-|
-| project | string | The GCP project hosting the Cloud Run service. | Yes |
-| region | string | The region of running Cloud Run service. | Yes |
-| credentialsFile | string | The path to the service account file for accessing Cloud Run service. | No |
-
-### CloudProviderLambdaConfig
-
-| Field | Type | Description | Required |
-|-|-|-|-|
-| region | string | The region of running Lambda service. | Yes |
-| credentialsFile | string | The path to the credential file for logging into AWS cluster. If this value is not provided, piped will read credential info from environment variables. It expects the format [~/.aws/credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html). | No |
-| roleARN | string | The IAM role arn to use when assuming an role. Required if you want to use the AWS SecurityTokenService. | No |
-| tokenFile | string | The path to the WebIdentity token the SDK should use to assume a role with. Required if you want to use the AWS SecurityTokenService. | No |
-| profile | string | The profile to use for logging into AWS cluster. The default value is `default`. | No |
-
-### CloudProviderECSConfig
-
-| Field | Type | Description | Required |
-|-|-|-|-|
-| region | string | The region of running ECS cluster. | Yes |
-| credentialsFile | string | The path to the credential file for logging into AWS cluster. If this value is not provided, piped will read credential info from environment variables. It expects the format [~/.aws/credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) | No |
-| roleARN | string | The IAM role arn to use when assuming an role. Required if you want to use the AWS SecurityTokenService. | No |
-| tokenFile | string | The path to the WebIdentity token the SDK should use to assume a role with. Required if you want to use the AWS SecurityTokenService. | No |
-| profile | string | The profile to use for logging into AWS cluster. The default value is `default`. | No |
+This field is deprecated, please use [PlatformProvider](#platformprovider) instead.
```

Comment:
This seems not to be necessary because we already mention this.
https://github.com/pipe-cd/pipecd/pull/4082/files#diff-cda66baafdfb61c1ac72ad0820325e806da1e2608cc9f64069b7b84abe967abbR33